environment/src/main/java/com/sequenceiq/environment/environment/service/freeipa/FreeIpaService.java

```diff
@@ -229,4 +239,19 @@ void stopFreeIpa(String environmentCrn) {
             throw new FreeIpaOperationFailedException(errorMessage, e);
         }
     }
+
+    public void rotateSaltPassword(EnvironmentDto environmentDto) {
+        String userCrn = ThreadBasedUserCrnProvider.getUserCrn();
+        try {
+            LOGGER.info("Rotating salt password of FreeIpa cluster");
+            freeIpaV1Endpoint.rotateSaltPassword(environmentDto.getResourceCrn());
```

Comment:
please use internal call here to skip authz (since call already authorized during environment call).
example: https://github.com/hortonworks/cloudbreak/blob/c9540968c2c0b3178c31bffe763431d72bd77cd6/datalake/src/main/java/com/sequenceiq/datalake/service/sdx/ProvisionerService.java#L81